crates/ide-assists/src/handlers/extract_function.rs

```diff
@@ -91,6 +92,8 @@ pub(crate) fn extract_function(acc: &mut Assists, ctx: &AssistContext) -> Option
 
     let scope = ImportScope::find_insert_use_container(&node, &ctx.sema)?;
 
+    let name = make_function_name(&semantics_scope);
```

Comment:
This can be moved into the builder closure

Reply:
Done!